### ISSUE TITLE
I install all the requirement as ur recommendation, but it fails

### ISSUE THREAD

**Issue by yuchenlichuck** [cure-lab/MotionCraft#3]
Pytorch3d doesn't have any match verision

**Comment by yuchenlichuck** [comment]
It works but I just realize u can generate long dance video, is that true?

**Comment by yxbian23** [comment]
> It works but I just realize u can generate long dance video, is that true?

Yeah, as our demo shows, we can generate relatively long videos by leveraging smooth denoising (https://arxiv.org/abs/2201.09865).
